cves/kernel/CVE-2021-30002.yml

```
@@ -375,8 +376,10 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    everything happens in the same order, but now it uses a pointer for an
```

Make sure to capitalize your sentences :) also making ambiguous language like "it" more specific throughout the document would be good
```suggestion
    Everything happens in the same order, but now the driver uses a pointer for an
```